modules/appeals_api/spec/workers/higher_level_review_pdf_submit_job_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'rails_helper'
 require AppealsApi::Engine.root.join('spec', 'spec_helper.rb')
+require AppealsApi::Engine.root.join('spec', 'support', 'shared_examples_for_monitored_worker.rb')
```

Comment:
I thought the spec helper already automatically included the support files? I'm not sure we have to add this line to get the monitored worker behaviors.

Reply:
It didn't work without it, but i'll take another look!